ajax/libs/iris-ws/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "iris-ws",
+  "title": "iris-ws",
+  "filename": "client.js",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/kataras/iris.git",
+    "fileMap": [
+      {
+        "basePath": "websocket",
+        "files": [
+          "client.js",
+          "client.min.js"
+        ]
+      }
+    ]
+  },
+  "description": "Websocket client-side library for the golang web-framework called iris.",
+  "author": {
+    "name": "Gerasimos Maropoulos",
+    "email": "kataras2006@hotmail.com"
+  },
+  "keywords": [
+    "iris",
+    "golang",
+    "websocket"
```

Comment:
Maybe some more keywords from the upper plane of it git repo?

Reply:
added a few more